src/Traveler.js

```diff
@@ -22,6 +20,17 @@ class Traveler {
     }, [])
   }
 
+  getCurrentTrip() {
+    return this.tripHistory.reduce((acc, trip) => {
+      let tripDate = trip.date;
+      let tripEndDate = moment().add(trip.duration, 'day').format("YYYY/MM/DD")
+      if (moment(tripDate).isBetween(tripDate, tripEndDate) && trip.travelerName.id === this.id) {
+        acc.push(trip)
+      }
+      return acc
+    }, [])
```

Comment:
Don't forget to change 'acc' to a variable name! Will will get mad haha. Maybe call it "currentTrips" here